docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,284 @@
+    using Tizen.Network.Nlp;
+    ```
+
+<a name="word_tokenize_result"></a>
+## To get the tokens of sentence 
+
+To recieve the tokens from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the WordTokenizeAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.WordTokenizeAsync(msg);
+        WordTokenizeResult wr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="pos_Tag_result"></a>
+## To get the tokens and tags of sentence 
+
+To recieve the tokens and tags from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the PosTagAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnPosButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.PosTagAsync(msg);
+        PosTagResult pr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="ne_chunk_result"></a>
+## To get the tokens and tags of sentence 
+
+To recieve the tokens and tags from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the NamedEntityRecognitionAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.NamedEntityRecognitionAsync(msg);
+        NamedEntityRecognitionResult nr = await ret;
+        ..
+    }
+    ```
+
+3.  When it is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+
+
+<a name="lemmatize_result"></a>
+## To get the actual word of sentence 
+
+To recieve the actual word from sentence:
+
+1.  Construct an NaturalLanguageProcess object on OnCreate(), and Connect the Nlp service on init of app :
```

Comment:
Construct NaturalLanguageProcess object on OnCreate(), and connect the Nlp service on init of app: